### ISSUE TITLE
Lightning example: double augmentation

### ISSUE THREAD

**Issue by sorenrasmussenai** [lucidrains/byol-pytorch#21]
Thanks for sharing this!

It seems to me that the Pytorch Lightning example augments the images twice; once in examples/lightning/train.py using Torchvision transforms and once in the BYOL class using Kornia augmentations.

/Søren

**Comment by lucidrains** [comment]
@sorenrasmussenai Hi Søren! So the torchvision transform only makes sure that what is fed into the network is a square with the right dimensions, in the case that your training set has rectangular images not of proper size.

**Comment by JayanthRR** [comment]
@lucidrains True what you pointed out. However, I observed that the performance of the algorithm (I was measuring clustering accuracy of the embeddings) degrades upon performing the augmentation twice. Since you are performing a resize, and center crop (while creating the dataset), the effective area of the image visible to the augmentations such a random resized crop (using kornia) is limited. If I added  random resized crop as a part of torchvision transforms while creating the dataset, and performed the other augmentations in kornia, I saw a boost in performance.
